bindings/nodejs/tests/types/protocol_parameters.spec.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { plainToInstance, Type } from 'class-transformer';
+import { describe, it } from '@jest/globals';
+import { expect } from '@jest/globals';
+import * as protocol_parameters from '../../../../sdk/tests/types/fixtures/protocol_parameters.json';
+import { ProtocolParameters, WorkScoreParameters, ManaParameters } from '../../lib/types/models/info/node-info-protocol';
+import { StorageScoreParameters } from '../../lib/types/models';
+
+describe('ProtocolParameters tests', () => {
+
+    it('creates ProtocolParameters from a fixture', async () => {
+
+        // TODO: is there a way around `as unknown`? The string->bigint parameters are the issue.
+        const params = protocol_parameters.params as unknown as ProtocolParameters;
+
+        // Check whether parameters were correctly set
+        expect(params.type).toEqual(0);
+        expect(params.version).toEqual(3);
+        expect(params.networkName).toEqual("TestJungle");
+        expect(params.bech32Hrp).toEqual("tgl");
```

Comment:
I don't think we need to compare all the fields btw, we just need to compare the hash and it'll automatically include all the fields

Reply:
Fixed